pkgs/sketch_pad/lib/execution/execution.dart

```diff
@@ -48,8 +48,13 @@ html.Element _iFrameFactory(int viewId) {
 class ExecutionWidget extends StatefulWidget {
   final AppServices appServices;
 
+  /// Whether the iframe ignores pointer events, for when gestures need to be
+  /// handled by the Flutter app.
+  final bool ignorePointer;
+
   ExecutionWidget({
     required this.appServices,
+    this.ignorePointer = false,
```

Comment:
Should we be passing on some sort of a ValueListenable here, and have the state's build method listen for changes to that value? The app's model could then have a something like a 'ValueListenable<bool> get enableExecutionServiceMouseEvents` or similar.

Or, the `ExecutionService` class, instead of exposing an html iFrame through its interface, could expose an setter to toggle enabling and disabling mouse events?

Reply:
👍